.github/.codecov.yml

```diff
@@ -37,15 +37,6 @@ flag_management:
           target: 20%
         - type: patch
           target: 100%
-    - name: nucliadb
-      paths:
-        - nucliadb/**
-      carryforward: true
-      statuses:
-        - type: project
-          target: 20%
-        - type: patch
-          target: 80%
```

Comment:
Why have you deleted it? Shouldn't it be `nucliadb/nucliadb/**`?

Reply:
Because "general rules" is managed by the "default_rules" key from a bit above. 
Essentially, this was making most of PRs show with the red cross